src/components/utils/file.js

```diff
@@ -1,4 +1,4 @@
-import loadImage from 'blueimp-load-image/js/load-image'
+import loadImage from 'blueimp-load-image/js'
```

Comment:
There is an [index.js](https://github.com/blueimp/JavaScript-Load-Image/blob/master/js/index.js) so node projects can import everything in one statement.

This protects us from future changes and means can use any version.